packages/quantic/package.json

```diff
@@ -24,9 +24,9 @@
     "test:unit:watch": "lwc-jest --watch",
     "test:unit:coverage": "lwc-jest --coverage",
     "validate:types": "tsc",
-    "e2e:watch": "npx cypress open",
-    "e2e": "npx cypress run --browser chrome",
-    "e2e:detailed": "npx cypress run --browser chrome --headless --reporter cypress/reporters/detailed-reporter.js",
+    "e2e:watch": "cypress open --browser chrome --e2e",
```

Comment:
I personally liked that we didn't specify chrome directly in the command in order to get the ability to easily switch the browser where the tests will be executed directly from the UI when I'm watching the tests locally:

<img width="1767" alt="e2e:watch" src="https://user-images.githubusercontent.com/86681870/236403472-f861b156-4b55-4d3d-bdad-dbe9b251bf00.png">

Reply:
It is still possible to switch to a different browser - this just launches cypress directly into choosing the test suite dialog - its just one click away to get back to choose a browser dialog.